source/declarators.tex

```diff
@@ -2388,11 +2388,11 @@
 \indextext{initialization|(}
 
 \pnum
-A declarator can specify an initial value for the
+An \grammarterm{init-declarator} can specify an initial value for the
```

Comment:
> A declarator can be followed by an initializer that specifies an initial value for the entity being declared.

This is description is correct, however I think we should emphasize on the 'initializer' which is the topic of this section [dcl.init]

If we look at the introductory description of [dcl.decl] p1 
> The init-declarator-list appearing in a declaration is a comma-separated sequence of 
declarators, each of which can have an initializer.

It is easy to understand [dcl.init] was intended for "initializer".